.github/workflows/reusable-deploy-api.yml

```diff
@@ -24,6 +24,8 @@ on:
         required: true
       LOGTAIL_TOKEN:
         required: true
+      UCAN_LOG_BASIC_AUTH:
```

Comment:
can this include a description of the format? e.g. in the past I've wondered if it should be base64 encoded or not. If not commented, I assume yes? (i.e. I'd assume it's the value in an http header after 'Basic '.

Reply:
Value after Basic encoded as base64